added release notes for 10.16.0.383 (#342)
SoftwareAG/c8y-releasenotes: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-383.md

---
weight: 25
title: Release 10.16.0.383
layout: bundle
---

<!--10.16.0.377-10.16.0.383-->

### Fixes

<div><table ><colgroup>
<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
<thead><tr>
<th>
Component</th>
<th>
Description</th>
<th>
Issue</th>
</tr>
</thead><tbody>

<tr>
<td>Device management</td>

> <td>Availability calculations in the "Availability" overview and for individual devices were not correct. Percentages were calculated for the needed time plus an additional 24 hours. The time interval has now been corrected.</td>

Reviewer: ```suggestion
<td>Availability calculations in the "Availability" overview and for individual devices were not correct. Percentages were calculated for the needed time plus an additional 24 hours. The time interval has now been corrected.</td>
```

what's the correct interval?

maybe also replace "needed" with "required"

Author: Changed to be more clear

> <td>Availability calculations in the "Availability" overview and for individual devices were not correct. Percentages were calculated for the needed time plus an additional 24 hours. The time interval has now been corrected.</td>

Author: ```suggestion
<td>The availability calculations in the "Availability" overview and for individual devices were not correct. Percentages were calculated for the respective time (24 hours, last 7 days and last 30 days) plus an additional 24 hours. The time interval has now been corrected.</td>
```

<td>DM-2737</td>
</tr>

<tr>
<td>Device management</td>

> <td>If editing a device protocol details page and navigating out of that page, a confirmation popup is now always properly shown.</td>

Reviewer: ```suggestion
<td>After editing a device protocol details page and leaving the page, a confirmation pop-up is now always properly shown.</td>
```
